src/core/ev.c

```diff
@@ -2952,10 +2952,15 @@ JANET_CORE_FN(cfun_ev_sleep,
 
 JANET_CORE_FN(cfun_ev_deadline,
               "(ev/deadline sec &opt tocancel tocheck)",
-              "Set a deadline for a fiber `tocheck`. If `tocheck` is not finished after `sec` seconds, "
-              "`tocancel` will be canceled as with `ev/cancel`. "
-              "If `tocancel` and `tocheck` are not given, they default to `(fiber/root)` and "
-              "`(fiber/current)` respectively. Returns `tocancel`.") {
+              "Schedules the event loop to try to cancel the `tocancel` "
+              "task as with `ev/cancel`.  After `sec` seconds, the event "
+              "loop will attempt cancellation of `tocancel` if the "
+              "`tocheck` fiber is resumable.  `sec` is a number that can "
```

Comment:
> resumable.  `sec`

should become

> resumable. `sec`

Reply:
Thanks for noticing.

I think I found a third one too (^^;

Since https://github.com/janet-lang/janet/commit/5de889419ff26b710b706958bf99e180d084f564 seems to suggest that lengthier docstrings might become an issue, I've removed the extra spaces.

I suspect there may be other cases in other docstrings.